src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$blockSize: 100px;
+$gapSize: 1px;
+$weekDays:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
+
+.calendar {
+  position: absolute;
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: $blockSize;
```

Comment:
It's up to you, but I'd recommend just to center the calendar instead of setting paddings